app/src/pages/inside/stepPage/modals/makeDecisionModal/elements/testItemDetails/testItemDetails.jsx

```diff
@@ -0,0 +1,139 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames/bind';
+import PropTypes from 'prop-types';
+import Parser from 'html-react-parser';
+import commentIcon from 'common/img/comment-inline.svg';
+import asteriskIcon from 'common/img/asterisk-inline.svg';
+import { StackTraceMessageBlock } from 'pages/inside/common/stackTraceMessageBlock';
+import { uniqueId } from 'common/utils';
+import { ERROR_LOGS_SIZE } from '../../constants';
+import { ItemHeader } from '../itemHeader';
+import styles from './testItemDetails.scss';
+
+const cx = classNames.bind(styles);
+
+export const TestItemDetails = ({
+  item,
+  mode,
+  selectItem,
+  logs,
+  eventsInfo,
+  highlightedLogId,
+  highlightedMessage,
+  noLogsMessage,
+  hideLabels,
+  onClickLinkEvent,
+  isSelected,
+  showErrorLogs,
+}) => {
+  const [showDetails, setShowDetails] = useState(showErrorLogs);
+
+  useEffect(() => {
+    setShowDetails(showErrorLogs);
+  }, [showErrorLogs]);
+
+  const toggleDetails = () => {
+    setShowDetails(!showDetails);
+  };
+
+  return (
+    <div className={cx('test-item-details')}>
+      <ItemHeader
+        item={item}
+        mode={mode}
+        selectItem={selectItem}
+        isSelected={isSelected}
+        hideLabels={hideLabels}
+        onClickLinkEvent={onClickLinkEvent}
+        toggleDetails={toggleDetails}
+      />
+      {showDetails && (
+        <div className={cx('test-item-details-content')}>
+          {item.issue && item.issue.comment && (
+            <div className={cx('defect-comment')}>
+              <div>{Parser(commentIcon)}</div>
+              <div className={cx('defect-comment-content')}>
+                <p className={cx('defect-comment-title')}>Defect Comment</p>
+                <p className={cx('defect-comment-text')}>{item.issue.comment}</p>
+              </div>
+            </div>
+          )}
+          {showDetails &&
+            logs.slice(0, ERROR_LOGS_SIZE).map((log) => (
+              <div key={uniqueId()} className={cx('error-log')}>
+                <StackTraceMessageBlock
+                  level={log.level}
+                  designMode="dark"
+                  maxHeight={90}
+                  eventsInfo={eventsInfo}
+                >
+                  <div className={cx('highlighted-log')}>
+                    {highlightedLogId === log.id && (
+                      <p className={cx('highlighted-log-title')}>
+                        {Parser(asteriskIcon)} {highlightedMessage}
+                      </p>
+                    )}
+                    {log.message}
+                  </div>
+                </StackTraceMessageBlock>
+              </div>
+            ))}
+          {showDetails && !logs.length && (
+            <div className={cx('no-logs')}>
+              <p className={cx('no-logs-text')}>{noLogsMessage}</p>
```

Comment:
This is no need to provide this message from props, you can specify it directly here.